### ISSUE TITLE
xclip erroring unexpectedly

### ISSUE THREAD

**Issue by mdlincoln** [mdlincoln/clipr#28]
```
  > test_check("clipr")
  1. Failure: clipr_available fails when DISPLAY is not configured; succeeds when it is (@test_render.R#12) 
  `is_clipr_available` isn't true.
  
  
  testthat results ================================================================
  OK: 1 SKIPPED: 11 FAILED: 1
  1. Failure: clipr_available fails when DISPLAY is not configured; succeeds when it is (@test_render.R#12) 
  
  Error: testthat unit tests failed
  Execution halted
```

**Comment by mdlincoln** [comment]
Travis is passing now? Unclear what caused this to begin with... equally unclear why it's solved.

**Comment by mdlincoln** [comment]
Hm, no, it appears that an uncached build on Travis fails, but not for xsel - just for xclip.

**Comment by mdlincoln** [comment]
This is insane. Apparently, xclip now needs to be coaxed into having a clipboard by first calling it before running the script.

**Comment by mdlincoln** [comment]
Closed with a02c4f5a
